etc/kayobe/ansible/octavia-amphora-image-build.yml

```diff
@@ -89,15 +90,15 @@
         - name: Create the Amphora image
           ansible.builtin.shell:
             cmd: >-
-              source {{ venv_path }}/bin/activate &&
+              . {{ venv_path }}/bin/activate &&
```

Comment:
What's the issue with source here?

Reply:
I'm not completely sure but the workflow was failing to find the command "source"